AABB_tree/doc/AABB_tree/Concepts/AABBGeomTraits_2.h

```diff
@@ -0,0 +1,202 @@
+
+/*!
+\ingroup PkgAABBTreeConcepts
+\cgalConcept
+
+The concept `AABBGeomTraits_2` defines the requirements for the first template parameter of the class
+`CGAL::AABB_traits_2<AABBGeomTraits_2, AABBPrimitive>`. It provides predicates and constructors to detect
+and compute intersections between query objects and the primitives stored in the AABB tree.
+In addition, it contains predicates and constructors to compute distances between a point query
+and the primitives stored in the AABB tree.
+
+\cgalRefines{SearchGeomTraits_2}
+
+\cgalHasModelsBegin
+\cgalHasModelsBare{All models of the concept `Kernel`}
+\cgalHasModelsEnd
+
+\sa `CGAL::AABB_traits_2<AABBGeomTraits_2,AABBPrimitive>`
+\sa `CGAL::AABB_tree<AABBTraits>`
+\sa `AABBPrimitive`
+
+*/
+
+class AABBGeomTraits_2 {
```

Comment:
Check CGAL for `AABBGeomTraits`, there are still many iterations in for example PMP, Surface_mesh_segmentation.